vmdb/app/views/shared/summary/_textual_multilabel.html.haml

```diff
@@ -11,5 +11,9 @@
     - items[:values].each do |item|
       %tr
         - item.each do |value|
-          %td
-            = value
+          - if value.is_a?(Hash)
```

Comment:
Where are the uses of the old options?  If it's just in the SummaryHelpers, maybe we should just change them?